src/Tabs.jsx

```diff
@@ -0,0 +1,31 @@
+import React from 'react';
+
+export class Tabs extends React.Component {
+  state = {}
```

Comment:
if you don't need state, it's better to use functional component than a class component